views/scores.pug

```diff
@@ -0,0 +1,8 @@
+html
+  head
+    title High Scores - #{gameName}
+  body
+    h1 High Scores - #{gameName}
+    div
+      each score in scores
+        div= gameName + ': ' score.score + ' - ' + score.name + ' at ' + score.datetime
```

Comment:
I think it'd be a little repetitive for the game name to show up on every line of scores, it might be better to only have it at the top.

Reply:
I was thinking it would be useful if we were to show all the high scores for a user in all games, but I guess it might be worth it to make a separate template for that or just worry about it later. Fixed in https://github.com/knative-portability/Kubercade/pull/5/commits/05985ee3fcb3cb68f9409dac427051762bba9055